### PR TITLE
[7.4.x only] LPS-121966 Replacing clay-vertical-card-v2 with clay-vertical-card

### PR DESCRIPTION
**Pull request by carloslancha**
Replacing all usages of `<clay:vertical-card-v2>` with `<clay:vertical-card>`

Note that [this](https://github.com/liferay-echo/liferay-portal/pull/2991/commits/c7300e1c99a46024fcd1f2d8fd724d1ec912dd7d) and [this](https://github.com/liferay-echo/liferay-portal/pull/2991/commits/2a663a80be6399caf9695ec9aa46f0e7bfa9e105) usages are meant to be using `navigation-card`, so we highly recommend to replace it with `<clay:navigation-card>` once https://issues.liferay.com/browse/LPS-123370 is completed.

Here we're also using `propsTransformer` pattern to replace old `defaultEventHandler` and other usages like manually added handlers using `dom.delegate`. We're doing this replacement in this pr as a base and example for you to replace other usages in your applications. In followings prs we'll be migrating `<clay:*-card>` to the new tags but the rest of this `propsTransformer` migration is up to you.

Don't hesitate to reach us to solve any doubt 😃

### CODE REVIEW

**Comment by carloslancha** [comment]
cc @julien @markocikos

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
Hi @victorg1991, I found a regression issue caused by this PR.
Add a master page template
Back to Page Templates admin
When click the ellipsis button of master page template, the following error is thrown in browser.
BrowserError

**Comment by carloslancha** [comment]
There's a typo in `propsTransformer="js/propsTransformer/MasterLayoutDropdownPropsTransformer"` (an `s` is missing in `js/propsTransformers`) and a missing actions safe check. Fixing it. Thanks @Tim-Cao

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
This is happening with the changes included here: blue_check
Note the blue icon flickering

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	},
+	portletNamespace: namespace,
+	...otherProps
+}) {
+	return {
+		...otherProps,
+		onClick: (event) => {
+			event.preventDefault();
+
+			openSimpleInputModal.default({
```
Comment (victorg1991) [inline modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer.js:33]: This props trasnformer is failing due to the extra `.default` here

Comment (carloslancha) [reply modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer.js]: Fixed

**Comment by carloslancha** [comment]
The issue with the flickering icon is solved in other pr, I've chery-picked that commit so it works fine here too

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:parate:plis

**Comment by victorg1991** [comment]
ci:forward

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**